### DISCUSSION TITLE
Welcome to argouml Discussions!

### DISCUSSION THREAD

**Discussion by linustolke** [argouml-tigris-org/argouml#60] [Announcements]
## 👋 Welcome!
Trying to embrace the full set of community tools available at github we have now enabled Discussions as a place to connect with each-other. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

These discussions are visible to anyone, even not logged-in users on github. To write new issue and comment you need to log in. 

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alepekhin** [comment]
Hi, I am a former developer of ArgoUML
What I expect from ArgoUML - is to be a simple free drawing UML tool 
I need to draw diagrams - not very strict to UML version - diagrams that I can show my colleagues to discuss an follow
I think in now days nobody needs to convert diagrams to the code or do reverse engineering except fully beginners
So I suggest to move aside initial concept and concentrate on drawing only
